tools/sof_perf_analyzer.py

```diff
@@ -27,8 +27,13 @@
 
 import pandas as pd
 
-# currently, keep CPC = max(module peak) * CPC_MARGIN
-CPC_MARGIN = 1.1
+# CPC_MARGIN set to 1.5 because some modules have not running code, for example
+# volume: volume have two kind of ramp type, linear and log, it only be runned when
+# volume change happens.
+# copier: there is attenuation inside, normally, it will not be runned.
+# So we set a relative high margin for avg cycles.
+# CPC = AVG(module) * CPC_MARGIN
```

Comment:
passive voice for run is run, not runned.
I don't think copier example is a good one, I don't think attenuation will impact the `copier_process` ops

```
CPC_MARGIN is set to 1.5, because there is some inactive code for some module
due to unmet condition. For example:
volume: ramp operation only run on volume change.

So, we set a relative high margin for avg cycles.
CPC = AVG(module) * CPC_MARGIN
```

Reply:
for copier, there are three main part, copier frame work, copier copy, and attenuation, it is a suitable case for set margin to 1.5.

Anyway, I don't care which case is suitable or not, it is in my mind, I changed based on your suggestion.